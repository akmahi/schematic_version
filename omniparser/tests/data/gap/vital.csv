"ROW1","Domain","Patient First Name","Patient Last Name","Patient Date of Birth","External Patient ID","External Practice ID","Vital Code","Vital Name","Vital Dictionary Name","Date Performed","Date Ordered","Date Due","Value","NPI","TIN","Provider Name","Record Verified","Medi Unnecessary Reason Code","Medi Unnecessary Reason Name","Medi Unnecessary Reason Dict","Rem Specified Vital Sign Flag","Vendor","Source Type","PATIENT_ID"
"1","Vital Signs","Georgiana","Abesamis","20121211","9509237","Epic","5","Blood Pressure Diastolic","","20230101","","","67","","","","","","","","","","","AHD6EGLLRG3BD3MEQSG3LJANY45GHANKYBTT5KXC"
"2","Vital Signs","Julian","Adamkiewicz","20120514","9763007","Epic","5","Blood Pressure Diastolic","","20230101","","","70","","","","","","","","","","",""
"3","Vital Signs","Ashley","Adams","19901228","7723132","Epic","5","Blood Pressure Diastolic","","20230101","","","72","","","","","","","","","","","AF35ENRRYDWBD2ELCN53FT6CDN4XCUDAP5MHNN23"
"4","Vital Signs","Dianne","Adams","19530815","4103210","Epic","5","Blood Pressure Diastolic","","20230101","","","70","","","","","","","","","","","AGIDOOACHHJRDZ47BLRZYHA2PBGFMHU47LXFTSUQ"
"5","Vital Signs","Marion","Adams","19800728","6149479","Epic","5","Blood Pressure Diastolic","","20230101","","","77","","","","","","","","","","","AFXJTYNQK3UBD2EYB53QTTFHONEBPF6JRBMBGXT2"
"6","Vital Signs","Toni","Adams","20061010","7784422","Epic","5","Blood Pressure Diastolic","","20230101","","","68","","","","","","","","","","",""
"7","Vital Signs","Emory","Ahart","20160504","6128058","Epic","5","Blood Pressure Diastolic","","20230101","","","60","","","","","","","","","","",""
"8","Vital Signs","Leo","Akroush","19870317","6291339","Epic","5","Blood Pressure Diastolic","","20230101","","","75","","","","","","","","","","","AEVV65U3P6BRD2F4CBO6QMLUPFCVF2DJPQOMBON2"
"9","Vital Signs","Patricia","Albano","19590222","2652897","Epic","5","Blood Pressure Diastolic","","20230101","","","76","","","","","","","","","","","AEH6P4FQW7GBD2EGFAIYUNX4L3BTAWA652LPZQSO"
"10","Vital Signs","Ava","Alexander","20090218","7803152","Epic","5","Blood Pressure Diastolic","","20230101","","","70","","","","","","","","","","",""
