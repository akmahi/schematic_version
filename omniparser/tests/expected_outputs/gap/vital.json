[
	{
		"vital": {
			"code": "5",
			"date_performed": "20230101",
			"domain": "Vital Signs",
			"name": "Blood Pressure Diastolic",
			"patient": {
				"dob": "20121211",
				"external_id": "9509237",
				"first_name": "Georgiana",
				"id": "AHD6EGLLRG3BD3MEQSG3LJANY45GHANKYBTT5KXC",
				"last_name": "Abesamis"
			},
			"practice": {
				"external_id": "Epic"
			},
			"row": "1",
			"value": "67"
		}
	},
	{
		"vital": {
			"code": "5",
			"date_performed": "20230101",
			"domain": "Vital Signs",
			"name": "Blood Pressure Diastolic",
			"patient": {
				"dob": "20120514",
				"external_id": "9763007",
				"first_name": "Julian",
				"last_name": "Adamkiewicz"
			},
			"practice": {
				"external_id": "Epic"
			},
			"row": "2",
			"value": "70"
		}
	},
	{
		"vital": {
			"code": "5",
			"date_performed": "20230101",
			"domain": "Vital Signs",
			"name": "Blood Pressure Diastolic",
			"patient": {
				"dob": "19901228",
				"external_id": "7723132",
				"first_name": "Ashley",
				"id": "AF35ENRRYDWBD2ELCN53FT6CDN4XCUDAP5MHNN23",
				"last_name": "Adams"
			},
			"practice": {
				"external_id": "Epic"
			},
			"row": "3",
			"value": "72"
		}
	},
	{
		"vital": {
			"code": "5",
			"date_performed": "20230101",
			"domain": "Vital Signs",
			"name": "Blood Pressure Diastolic",
			"patient": {
				"dob": "19530815",
				"external_id": "4103210",
				"first_name": "Dianne",
				"id": "AGIDOOACHHJRDZ47BLRZYHA2PBGFMHU47LXFTSUQ",
				"last_name": "Adams"
			},
			"practice": {
				"external_id": "Epic"
			},
			"row": "4",
			"value": "70"
		}
	},
	{
		"vital": {
			"code": "5",
			"date_performed": "20230101",
			"domain": "Vital Signs",
			"name": "Blood Pressure Diastolic",
			"patient": {
				"dob": "19800728",
				"external_id": "6149479",
				"first_name": "Marion",
				"id": "AFXJTYNQK3UBD2EYB53QTTFHONEBPF6JRBMBGXT2",
				"last_name": "Adams"
			},
			"practice": {
				"external_id": "Epic"
			},
			"row": "5",
			"value": "77"
		}
	},
	{
		"vital": {
			"code": "5",
			"date_performed": "20230101",
			"domain": "Vital Signs",
			"name": "Blood Pressure Diastolic",
			"patient": {
				"dob": "20061010",
				"external_id": "7784422",
				"first_name": "Toni",
				"last_name": "Adams"
			},
			"practice": {
				"external_id": "Epic"
			},
			"row": "6",
			"value": "68"
		}
	},
	{
		"vital": {
			"code": "5",
			"date_performed": "20230101",
			"domain": "Vital Signs",
			"name": "Blood Pressure Diastolic",
			"patient": {
				"dob": "20160504",
				"external_id": "6128058",
				"first_name": "Emory",
				"last_name": "Ahart"
			},
			"practice": {
				"external_id": "Epic"
			},
			"row": "7",
			"value": "60"
		}
	},
	{
		"vital": {
			"code": "5",
			"date_performed": "20230101",
			"domain": "Vital Signs",
			"name": "Blood Pressure Diastolic",
			"patient": {
				"dob": "19870317",
				"external_id": "6291339",
				"first_name": "Leo",
				"id": "AEVV65U3P6BRD2F4CBO6QMLUPFCVF2DJPQOMBON2",
				"last_name": "Akroush"
			},
			"practice": {
				"external_id": "Epic"
			},
			"row": "8",
			"value": "75"
		}
	},
	{
		"vital": {
			"code": "5",
			"date_performed": "20230101",
			"domain": "Vital Signs",
			"name": "Blood Pressure Diastolic",
			"patient": {
				"dob": "19590222",
				"external_id": "2652897",
				"first_name": "Patricia",
				"id": "AEH6P4FQW7GBD2EGFAIYUNX4L3BTAWA652LPZQSO",
				"last_name": "Albano"
			},
			"practice": {
				"external_id": "Epic"
			},
			"row": "9",
			"value": "76"
		}
	},
	{
		"vital": {
			"code": "5",
			"date_performed": "20230101",
			"domain": "Vital Signs",
			"name": "Blood Pressure Diastolic",
			"patient": {
				"dob": "20090218",
				"external_id": "7803152",
				"first_name": "Ava",
				"last_name": "Alexander"
			},
			"practice": {
				"external_id": "Epic"
			},
			"row": "10",
			"value": "70"
		}
	}
]
